styles.css

```diff
@@ -1393,3 +1344,109 @@ textarea {
     color: #b3bac5;
   }
 }
+
```

Comment:
Because you didn't style the tags, I don't have a line to reference;

- [ ] In order to match the Figma design, please style the top of the popup which contains the tags and the title

**Your desktop version:**
![image](https://user-images.githubusercontent.com/83596135/180837000-95143337-50dc-4cb3-80be-b03c26390dc4.png)

**Figma design - desktop version:**
![image](https://user-images.githubusercontent.com/83596135/180836898-729a57b7-0074-42bb-bb42-4ae4e48fc0e5.png)